comixed-app/src/main/java/org/comixedproject/ComiXedApp.java

```diff
@@ -52,7 +52,6 @@ public void onDestroy() {
     log.info("Closing application context");
     ((ConfigurableApplicationContext) this.applicationContext).close();
     log.info("Shutting down");
```

Comment:
Since we're removing the following line, let's also delete this one log.info() call.

Reply:
We should leave the line.
You will see now that after that line more will appear in the log(from Spring), so it should be there.

Reply:
Fair enough.